scripts/swebench_context_packets.py

```
@@ -0,0 +1,2187 @@
+    candidates.sort(key=lambda item: PROTOCOL._replayed_candidate_sort_key(item, relationships))
+    for ordinal, candidate in enumerate(candidates, 1):
+        candidate["acquisition_ordinal"] = ordinal
+        semantic, graph, total, hops = PROTOCOL._derived_candidate_score(candidate, relationships)
+        candidate.update(
+            semantic_component=semantic,
+            graph_component=graph,
+            total=total,
+            graph_hops=hops,
+        )
+    selected, omissions = PROTOCOL._replay_candidate_admission(candidates)
+    for candidate, is_selected in zip(candidates, selected):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)